receiver/apachereceiver/testdata/integration/expected.yaml

```diff
@@ -215,5 +215,5 @@ resourceMetrics:
                   timeUnixNano: "1632495518500962000"
             unit: '{workers}'
         scope:
-          name: otelcol/apachereceiver
+          name: otelcol/apache
```

Comment:
i wouldn't expect this to change unless we agree to it, this is a breaking change for users

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21382 is filed